src/azure_devtools/scenario_tests/tests/test_utilities.py

```diff
@@ -17,7 +17,7 @@ def test_create_random_name_default_value(self):
         self.assertTrue(isinstance(default_generated_name, str))
 
     def test_create_random_name_randomness(self):
-        self.assertEqual(100, len(set([create_random_name() for _ in range(100)])))
+        self.assertEqual(100, len({create_random_name() for _ in range(100)}))
```

Comment:
Is this on purpose?

Reply:
Yeah. It's a pylint suggestion, `consider-using-set-comprehension`